test/lib/ansible_test/_util/controller/sanity/validate-modules/validate_modules/schema.py

```diff
@@ -296,7 +296,7 @@ def argument_spec_schema(for_collection):
                 (is_callable, list_string_types),
                 [is_callable, list_string_types],
             ),
-            'choices': Any([object], (object,)),
+            'choices': Any([object], (object,), {object: Any[str,[str]]}),
```

Comment:
Ah sorry, I think I pointed you to the wrong `choices` :facepalm:. This one is the argspec, not the documentation.

The right place is here: https://github.com/ansible/ansible/blob/689c905c6e5dfbc67817ff779c02cce24be14f27/test/lib/ansible_test/_util/controller/sanity/validate-modules/validate_modules/schema.py
This should probably be `'choices': Any([json_value], {json_value: doc_string_or_strings]}),`